index.js

```diff
@@ -83,7 +127,18 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let movie = null;
+  if (!movies.length) {
+    return movie;
+  }
+  for (let i = 0; i < movies.length; i++) {
+    if (movies[i].imdbID === id) {
+      movie = movies[i];
```

Comment:
I would `return movie` here as soon as we find it.  In the case that we have 100,000 movies and we find what we're looking for at index 2, we currently have to loop through the whole array still.